edweiss-app/constants/Colors.ts

```diff
@@ -16,7 +16,7 @@ export type Color = keyof typeof Colors.light & keyof typeof Colors.dark;
 const Colors = {
 	light: {
 		transparent: '#0000',
-		// course_title_for_backgroud_color: "#000000",
+		course_title_for_backgroud_color: "#000000",
```

Comment:
I know that this file handle the color palette of the app. However, I don't know about it history. Can you explicit me why you had to de-coment this color.  I guess that it name is very explicit  😁, but I am interresting in the picture 🔮.

Reply:
This change I made was due to some tests not passing but it was resolved in a recent merge made to the main so I just fixed it now when handling merge conflicts. So there is no need to worry

Reply:
Ah great, I understand your strategy. Reduce at maximum the number of moving part so that you can better analyzing the moving parts. If I am not mistaken, it a good application of the `least privilege` principle, don't you think so.  (To be or not to be 💀, the question was not even intended to be asked 😋).